drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -34,7 +34,57 @@ static unsigned int nonce_cnt;
 static nrfx_qspi_config_t QSPIconfig;
```

Comment:
Not going to leave a change request but we do not use camel case in ncs or zephyr, we use snake case